pkg/apic/resourcePagination.go

```diff
@@ -0,0 +1,62 @@
+package apic
+
+import (
+	"encoding/json"
+	"strconv"
+
+	coreapi "github.com/Axway/agent-sdk/pkg/api"
+	"github.com/Axway/agent-sdk/pkg/apic/apiserver/clients/definitions/v1alpha1"
+	"github.com/Axway/agent-sdk/pkg/apic/apiserver/models/management/v1alpha1"
+	"github.com/Axway/agent-sdk/pkg/util/log"
+)
+
+func (c *ServiceClient) fubar() error {
```

Comment:
Don't think you wanted to leave this...

Reply:
*NOTE*  - sorry (guys), this was a DRAFT for a single point of pagination func that was supposed to be checked into a different branch. PLEASE ignore for now.